app/assets/stylesheets/scaffolds.scss

```diff
@@ -65,6 +65,15 @@ body {
   font-size: 18px;
 }
 
+/* Hide headers for screen readers */
```

Comment:
You can use `aria-hidden` on the headers and that should remove them from the accessibility tree.

Reply:
I think the intention is actually the opposite. Rather than hiding these headers from the Accessbility API, I want them visually gone, but available to people using assistive technologies.

Here is an example: https://www.accessibility-developer-guide.com/examples/headings/visually-hidden-headings/

We could use these headers in many more places on the site, but I'm trying to use them at minimum in the main landmarks.

Reply:
Although the comment is misleading now that I read it without context. I will change that 😬

Reply:
Ahh, gotcha. All good then.